src/main/java/com/puppycrawl/tools/checkstyle/TreeWalker.java

```diff
@@ -79,6 +83,12 @@ public final class TreeWalker extends AbstractFileSetCheck implements ExternalRe
     /** A factory for creating submodules (i.e. the Checks) */
     private ModuleFactory moduleFactory;
 
+    /** Control whether to skip files with Java parsing exceptions. */
+    private boolean skipFileOnJavaParseException;
+
+    /** Severity Level to log Java parsing exceptions when they are skipped. */
```

Comment:
This should match setter javadoc as I mentioned in other issue. It should start with "Specify".

Reply:
done